packages/testing/index.js

```diff
@@ -1,8 +1,8 @@
-// remove esline-disable when https://github.com/esm-bundle/chai/pull/61 is merged
-/* eslint-disable-next-line import/no-unresolved */
-import chai, { expect, should, assert } from '@esm-bundle/chai';
+import * as chai from 'chai';
```

Comment:
it's easy to import and export just the needed modules:
```
export { expect, should, assert } from 'chai':
```

Or export all chai modules

```
export * from 'chai';
```